src/Stryker.Core/Stryker.Core/Initialisation/InitialBuildProcess.cs

```diff
@@ -22,10 +22,22 @@ public InitialBuildProcess(IProcessExecutor processExecutor = null)
             _logger = ApplicationLogging.LoggerFactory.CreateLogger<InitialBuildProcess>();
         }
 
-        public void InitialBuild(string path, string projectName)
+        public void InitialBuild(bool fullFramework, string path, string projectName)
         {
             _logger.LogInformation("Starting initial build");
-            var result = _processExecutor.Start(path, "dotnet", $"build {projectName}");
+            ProcessResult result = null;
+            if (fullFramework)
+            {
+                // Build with MSBuild.exe
+                var msBuildPath = new MsBuildHelper().GetMsBuildPath();
+                result = _processExecutor.Start(path, msBuildPath, "");
```

Comment:
We should run nuget restore (not dotnet restore) on the project first, to be certain packages have been restored. I am not sure whether or not this will recursively restore packages for pre- package-reference projects.. We should not include them in the supported projects before we are sure.